airflow-core/src/airflow/api_fastapi/execution_api/app.py

```diff
@@ -132,6 +132,7 @@ def create_task_execution_api_app() -> FastAPI:
         description="The private Airflow Task Execution API.",
         lifespan=lifespan,
         api_version_parameter_name="Airflow-API-Version",
+        api_version_default_value=bundle.versions[0].value,
```

Comment:
I think the following would be better

```suggestion
        api_version_default_value=bundle.head_version,
```

Reply:
@kaxil , head_version is an object.   `api_version_default_value` needs to be a string or a func returning string. 

I just set it to head_version and its failing

Reply:
aah damn. you are right.

In that case this is fine.

Other option is: `bundle.reversed_version_values[0]` but that's just nit